aruco_opencv/src/aruco_tracker.cpp

```diff
@@ -159,6 +161,13 @@ class ArucoTracker : public rclcpp_lifecycle::LifecycleNode
     cam_info_retrieved_ = false;
 
     std::string cam_info_topic = image_transport::getCameraInfoTopic(cam_base_topic_);
+    if (cam_base_topic_.size() > 0 && cam_base_topic_[0] != '/' &&
+      cam_info_topic.size() > 0 && cam_info_topic[0] == '/')
+    {
+      // Remove leading slash to allow for relative camera_base_topic.
+      cam_info_topic = cam_info_topic.substr(1);
+    }
```

Comment:
The `getCameraInfoTopic` function assumes that the name is completely resolved which I missed when I developed this package. Instead of removing leading slashes, I propose resolving the image topic name here, similarly to [how it's done in image_transport package](https://github.com/ros-perception/image_common/blob/humble/image_transport/src/camera_subscriber.cpp#L119-L121).

This should make it also work when the base camera topic is remapped. I haven't tested it though.

```suggestion
    std::string image_topic = rclcpp::expand_topic_or_service_name(
      cam_base_topic_, this->get_name(), this->get_namespace());
    std::string cam_info_topic = image_transport::getCameraInfoTopic(image_topic);
```